recipes/tttrlib/build.sh

```
@@ -0,0 +1,26 @@
+mkdir b2 && cd b2
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)